sktime/forecasting/base/adapters/_pytorch.py

```diff
@@ -56,8 +63,6 @@ def _fit(self, y, fh, X=None):
         X : iterable-style or map-style dataset
             see (https://pytorch.org/docs/stable/data.html) for more information
         """
-        from sktime.forecasting.base import ForecastingHorizon
-
```

Comment:
the saving of `_fh` and `_y` seems problematic, this should be controlled in the base class

Reply:
Fixed by other PR